docs/proposals/deploy-csi-to-k3s.md

```diff
@@ -0,0 +1,54 @@
+# Proposal: Deploy CSI on K3s cluster 
+
+Last updated: 8.04.22
+
+
+## Abstract
+
+Deploy CSI on K3s cluster, test it work and add steps for manual patching scheduler 
+
+## Background
+
+Currently csi is compatible with Vanilla Kubernetes, RKE2, OpenShift and Kind, but only for test purpose. We want to deploy csi to new platform.  
+
+## Proposal
+
+Need to create next config file in directory `/var/lib/rancher/k3s/agent/pod-manifests`:
+```
+apiVersion: kubescheduler.config.k8s.io/v1beta1
+kind: KubeSchedulerConfiguration
+extenders:
+  - urlPrefix: "http://127.0.0.1:8889"
+    filterVerb: filter
+    prioritizeVerb: prioritize
+    weight: 1
+    enableHTTPS: false
+    nodeCacheCapable: false
+    ignorable: true
+    httpTimeout: 15s
+leaderElection:
+  leaderElect: true
+clientConnection:
+  kubeconfig: /var/lib/rancher/k3s/server/cred/scheduler.kubeconfig
+```
+For passing it to k3s server component use `--kube-scheduler-arg=config`
+
+
+## Compatibility
+
+There is no problem with compatibility
+
+## Implementation
+
+Tasks:
+1.	Create new config for scheduler 
+2.	Pass it to the k3s application server with appropriate flag
```

Comment:
Yes, you're right. Maybe, exist the third way to pass new config to kube-scheduler, but I can't find it. However, both methods are good but they are used in different situations.
1) The first one is executed when the k3s service is not yet running on the machine. This command will modify the default service script with the required configuration file and start the service. If the service is already started this command will not be able to modify it. It will try to run another one in parallel with the default configuration and run into the problem of running two servers on the same host.
2) The second one for changing running service configuration. After modification it doesn't reload itself and need to do it manually. 

@safronovD Is it necessary to add this thread to Rationale section of proposal?

Reply:
Thank you! Yes, you could add it to propose or rationale section